git_wrapper/remote.py

```diff
@@ -65,13 +66,13 @@ def fetch(self, remote="origin"):
             raise exceptions.ReferenceNotFoundException(msg)
 
         try:
-            remote.fetch()
+            remote.fetch(prune=prune, tags=tags, branches=branches)
```

Comment:
By this point, remote is a GitPython object, isn't it? 

```
(Pdb) type(remote)
<class 'git.remote.Remote'>
```

I don't think this can work, as the remote object's function wouldn't understand [any of these parameters](https://github.com/gitpython-developers/GitPython/blob/main/git/remote.py#L918)? We'd need to switch the arguments to the flags as understood by git-fetch, I believe.

Ideally to prove this works well, an [integration test](https://github.com/release-depot/git_wrapper/tree/master/integration_tests) would be created. Because a remote needs to be modified, this isn't quite straightforward but the trick used when [testing repo cloning](https://github.com/release-depot/git_wrapper/blob/master/integration_tests/conftest.py#L37) might help if we want to investigate this approach.

Reply:
Remote is a GitPython object, and the parameters I have right now are incorrect but the correct parameters look like this:
`remote.fetch(prune=prune,prune_tags=tags)`
Which leads to the following git command being executed when it's used in `git_tag_cleaner`.
`INFO:git.cmd:git fetch -v --prune --prune-tags origin`
GitPython isn't super thoroughly documented, but I found out that they allow that by searching around on the internet.

Reply:
Looks good!